styles/snippet.css

```diff
@@ -1,12 +1,11 @@
 
 pre{
-    white-space: pre-line;
     max-height: 350px;
     overflow-y: auto;
 }
 code{
+    padding: 0px !important;
     line-height: 2;
-    padding: 2rem !important;
```

Comment:
avoiding using important

Reply:
here the pre tag gives auto padding to over write that i have to give important